_docs/xcm/hrmp-channel.md

```diff
@@ -133,9 +133,26 @@ tokenDecimals: 10
 ```
 
 ## Turing Staging
-Parachain sovereign account: 5Ec4AhNzVAVwDR54oqgWQ5YSoPbxXathSrQqTmqkF5o7xe6y
+The below details can be viewed on [Rococo Subscan](https://rococo.subscan.io/parachain/2114).
+
+Sovereign Account (Rococo V1): `5Ec4AhNzVAVwDR54oqgWQ5YSoPbxXathSrQqTmqkF5o7xe6y`
+
+Sovereign Account (Turing Staging): `5Dt6dpkWPwLaH4BBCKJwjiWrFVAGyYk3tLUabvyn4v7KtESG`
```

Comment:
Does this line mean the Sovereign account on Turing Staging?

It should be `68gzVdFcwnNUcwiExdbioBwGcktc2EZP9KQ1zvdTQn9WVdmn`.

Reply:
`68gzVdFcwnNUcwiExdbioBwGcktc2EZP9KQ1zvdTQn9WVdmn` shows in Turing Staging as parachain 2114. We need to understand the difference between `68gzVdFcwnNUcwiExdbioBwGcktc2EZP9KQ1zvdTQn9WVdmn` and `5Dt6dpkWPwLaH4BBCKJwjiWrFVAGyYk3tLUabvyn4v7KtESG`